docs/reference/builder.md

```diff
@@ -496,10 +496,8 @@ specified network ports at runtime. `EXPOSE` does not make the ports of the
 container accessible to the host. To do that, you must use either the `-p` flag
 to publish a range of ports or the `-P` flag to publish all of the exposed ports.
 You can expose one port number and publish it externally under another number.
- 
-Docker uses exposed and published ports to interconnect containers using links
-(see [Linking containers together](../userguide/dockerlinks.md))
-and to set up port redirection on the host system when [using the -P flag](run.md#expose-incoming-ports).
+
+To set up port redirection on the host system, see [using the -P flag](run.md#expose-incoming-ports). Docker networks supports creating networks without the need to expose ports within the network, the [*Docker network overview*""](../userguide/networking/index.md)).
```

Comment:
Missing "see" or "refer to" before the "Docker network" link.

Also, please wrap